apps/editorial_features/components/eoy/client.coffee

```diff
@@ -147,44 +141,9 @@ module.exports.EoyView = class EoyView extends Backbone.View
     , {offset: '50%'}
 
   animateBody: (scrollTop) =>
-    boundaries = @getBodySectionBoundaries()
-    active = @closestSection(scrollTop, boundaries.top) - 1
-    $('.article-body section[data-section!="' + active + '"] .article-body--section').removeClass('active')
-    $active = $('.article-body section[data-section="' + active + '"]')
-    $active.find('.article-body--section').addClass('active')
-    @animateActiveSection($active, scrollTop)
-
-  animateActiveSection: ($active, scrollTop) =>
-    active = $active.data('section')
-    spacers = []
-    switch
-      when active == 1
-        spacers.push {spacer: $active.find('.article-body--section__cover-image .spacer').first(), gutter: 40}
-        spacers.push {spacer: $active.find('.spacer--article'), gutter: 20}
-      when active == 3
-        max = $active.find('.article-body--section__sub-text .spacer').parent().height()
-        $active.find('.article-body--section__sub-text .spacer').css('max-height', max)
-        spacers.push {spacer: $active.find('.article-body--section__sub-text .spacer'), gutter: 20}
-        spacers.push {spacer: $active.find('.spacer--article'), gutter: 20}
-      when active == 4
-        spacers.push {spacer: $active.find('.article-body--section__cover-image .spacer').first(), gutter: 40}
-        spacers.push {spacer: $active.find('.spacer--article'), gutter: 40}
-      when active == 5
-        spacers.push {spacer: $active.find('header .spacer--article'), gutter: 20}
-        spacers.push {spacer: $active.find('.article-body--section__text .spacer'), gutter: 20}
-      when active == 6
-        spacers.push {spacer: $active.find('header .spacer--article'), gutter: 20}
-        spacers.push {spacer: $active.find('.article-body--section__text .spacer'), gutter: 20}
-    for spacer in spacers
-      throttled = _.throttle(@verticalScrollBoundary, 30)
-      throttled(scrollTop, spacer.spacer, spacer.gutter)
-
-  verticalScrollBoundary: (scrollTop, spacer, gutter) =>
-    windowBottom = scrollTop + @windowHeight
-    initHeight = spacer.height
-    spacerHeight = windowBottom - $(spacer).parent().offset().top - gutter
-    if initHeight != spacerHeight.toFixed()
-      $(spacer).height(spacerHeight.toFixed())
```

Comment:
Simple is best :) Glad you got a chance to try animations and explore its limitations.